src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1661,6 +1662,13 @@ private function getErrorMessages(Exception $e)
                     'Admin.Orderscustomers.Notification'
                 ),
             ],
+
+            OrderConstraintException::class => [
+                OrderConstraintException::INVALID_CUSTOMER_MESSAGE => $this->trans(
+                    'The order message given is invalid',
```

Comment:
@LouiseBonnard There's a bug in PrestonBot right now preventing the detection of new wordings but I think this one is new.

Reply:
Noted, thanks @atomiix! @sowbiba, why could the order message be invalid? Are there special characters forbidden or length restriction?

Reply:
I'll update the [specs](https://github.com/PrestaShop/prestashop-specs/commit/002dd12effcceede6d725d9fa01c5dc75587f279) accordingly. ;-)

Reply:
Yes there are forbidden special characters like <> and {}

Reply:
Oops, too bad it was merged, I should have labeled the PR as _waiting for wording_ since the bot is broken. It could have been nice to add a more specified notification. Next time!

Reply:
Yeah, thank you :kissing_cat: